roles/pulp_webserver/defaults/main.yml

```diff
@@ -11,3 +11,7 @@ pulp_webserver_httpd_servername: '{{ ansible_facts.fqdn }}'
 # https://pulp.plan.io/issues/5995
 pulp_webserver_static_dir: "{{ pulp_user_home | regex_replace('\\/$', '') }}/pulpcore_static"
 pulp_webserver_tls_files_remote: false
+
+_pulp_webserver_tls_key: '{{ pulp_webserver_tls_key | default("") }}'
+_pulp_webserver_tls_cert: '{{ pulp_webserver_tls_cert | default("") }}'
+_pulp_webserver_tls_custom_ca_cert: '{{ pulp_webserver_tls_custom_ca_cert | default("") }}'
```

Comment:
Sorry, i was not clear enough. At this point there is no need to prefix the variable. Just give the "real" variable a default to be "".
Using the ` | length` later is imho ok. What do others( @pulp/installer ) think? Have you tried whether ` | bool` produces the expected answer on strings?

Reply:
> Have you tried whether | bool 

I did and no that wouldn't work